doc/sphinx/data_notes.rst

```diff
@@ -282,10 +282,10 @@ If the optimisation has finished successfully, the values in the last row should
 
 
 grads
----------
-The attribute ``grads`` contains the atom forces (gradients) coordinates as taken from the output file. This is an array of rank 3, with a shape (n,m,3) where n is 1 for a single point calculation and >=1 for a geometry optimisation and m is the number of atoms. The orientation of ``grads`` is equal to that of `atomcoords`_.
+-----
+The attribute ``grads`` contains the forces on the atoms, that is, the gradient of the energy with respect to atomic coordinates. ``grads`` is an array of rank 3, with dimensions `n x m x 3` where `n` is 1 for a single point calculation and `>=1` for a geometry optimisation, `m` is the number of atoms and the last dimension contains the x, y and z components of the gradient. The orientation of ``grads`` corresponds to that of `atomcoords`_.
```

Comment:
negative gradient

Reply:
Actually, that was my exact question. Is `grads` the forces (negative of the gradient) or the gradient? Sign convention matters a lot here.

Reply:
I ran a slightly compressed H2 in Gaussian and the forces point away from each other (negative of the gradient). I didn't have time to check other codes, but I think this would be a good test that would could add in the future.